src/modules/navigator/navigator.h

```diff
@@ -408,6 +412,8 @@ class Navigator : public ModuleBase<Navigator>, public ModuleParams
 
 	float _mission_cruising_speed_mc{-1.0f};
 	float _mission_cruising_speed_fw{-1.0f};
+	float _mission_vertical_up_speed_mc{-1.0f};
+	float _mission_vertical_down_speed_mc{-1.0f};
```

Comment:
We could document the unit & what this variable is about here as well?

Reply:
updated